data/strings/types_strings.txt

```diff
@@ -3481,11 +3497,13 @@
   [type.barrier.city_wall]
     en = City wall
     ru = Городская стена
+    ja = 城壁
     pl = Mur miejski
 
   [type.barrier.cycle_barrier]
     en = Cycle barrier
     ru = Ограждение, предотвращающее проезд велосипедов
+    ja = 自転車止め
```

Comment:
Please see type description:
https://wiki.openstreetmap.org/wiki/Tag:barrier%3Dcycle_barrier
https://wiki.openstreetmap.org/wiki/JA:Tag:barrier%3Dcycle_barrier

Is `自転車止め` correct word do describe such objects? I ask because I do not speak Japanese and machine translation of  `自転車止め` looks more like bicycle stop/bicycle parking. Just want to be sure everything is ok.

Reply:
Thank you for pointing it out. `バリカー` is more appropriate than `自転車止め`.